style.css

```diff
@@ -138,17 +163,30 @@ body {
 
 .menu-list {
   list-style: none;
-  display: none;
-  gap: 2.4rem;
-  justify-content: center;
   flex: 1;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #fff;
+  padding-top: 4rem;
+  padding-left: 20%;
+  padding-bottom: 4rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 3rem;
+  min-height: 75.5rem;
```

Comment:
- [ ] [OPTIONAL] kindly it is better to make the `height: 100vh` to take the whole height of the page.